brain/python/arrayHelpers.cpp

```diff
@@ -0,0 +1,259 @@
+/* Copyright (c) 2013-2016, Juan Hernando <juan.hernando@epfl.ch>
+ *
+ * This file is part of Brion <https://github.com/BlueBrain/Brion>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "arrayHelpers.h"
+#include "../types.h"
+#include "../neuron/types.h"
+
+#include <vmmlib/matrix.hpp>
+#include <vmmlib/quaternion.hpp>
+#include <vmmlib/vector.hpp>
+
+#include <numpy/_numpyconfig.h>
+#if NPY_API_VERSION >= 0x00000007
+#  define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
+#endif
+#include <numpy/arrayobject.h>
+
+using namespace boost::python;
```

Comment:
can we used namespace bp = boost::python please? I was always confused in the SDK wrappings to find which code was related to boost.